backends/qualcomm/quantizer/utils.py

```diff
@@ -179,6 +186,44 @@ def _derive_bias_qparams_fn(
     )
 
 
+def get_default_8bit_qat_proto(act_symmetric: bool = False) -> QuantizationConfig:
+
+    act_quantization_spec = QuantizationSpec(
+        dtype=torch.uint8,
+        qscheme=(
+            torch.per_tensor_symmetric if act_symmetric else torch.per_tensor_affine
+        ),
+        ch_axis=0,
+        observer_or_fake_quant_ctr=default_fake_quant,
+    )
+
+    weight_quantization_spec = QuantizationSpec(
+        dtype=torch.int8,
+        quant_min=torch.iinfo(torch.int8).min + 1,
+        quant_max=torch.iinfo(torch.int8).max,
+        qscheme=torch.per_tensor_symmetric,
+        ch_axis=0,
+        observer_or_fake_quant_ctr=FusedMovingAvgObsFakeQuantize.with_args(observer=MovingAverageMinMaxObserver),
```

Comment:
How about using MovingAveragePerChannelMinMaxObserver instead of MovingAverageMinMaxObserver, since we are going to do per-channel quantization for the weights?

FusedMovingAvgObsFakeQuantize.with_args(observer=MovingAveragePerChannelMinMaxObserver)

Reply:
Thanks for the advice! Sorry for late reply because I was OoO.
If it's possible may we keep this for simplicity now? Currently we are working on different kind of quant confings for existing test cases. Once we finish we will update the quanziter with QAT.